packages/modules/viewer-react/src/services/BaseInitializer.ts

```diff
@@ -31,10 +31,11 @@ import type { ElectronRendererAuthorization } from "@itwin/electron-authorizatio
 import { PresentationRpcInterface } from "@itwin/presentation-common";
 import { Presentation } from "@itwin/presentation-frontend";
 
+import { ViewerPerformance } from "..";
```

Comment:
prefer not to import from internal barrel

Reply:
updated